lib/l10n_utils/tests/test_template.py

```diff
@@ -78,7 +78,7 @@ def test_added_lang_files(self):
         request = type('request', (), {})()
         template.render({'request': request})
         assert request.langfiles == [
-            'dude', 'walter', 'navigation', 'download_button', 'main', 'footer']
```

Comment:
To avoid having to change this in future, perhaps we should add `DOTLANG_FILES` to the `override_settings` for this test class.